src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -0,0 +1,78 @@
+import Box from '@mui/material/Box';
+import Chip from '@mui/material/Chip';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import OutlinedInput from '@mui/material/OutlinedInput';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import { Theme, useTheme } from '@mui/material/styles';
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+    PaperProps: {
+        style: {
+            maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+            width: 250,
+        },
+    },
+};
+
+function getStyles(name: string, selectedOptions: readonly string[], theme: Theme) {
+    return {
+        fontWeight:
+            selectedOptions.indexOf(name) === -1
+                ? theme.typography.fontWeightRegular
+                : theme.typography.fontWeightMedium,
+    };
+}
+
+interface MultipleSelectProps {
+    options: string[];
+    selectedOptions: string[];
+    setSelectedOptions: (options: string[]) => void;
+}
+
+const MultipleSelect = ({ options, setSelectedOptions, selectedOptions = [] }: MultipleSelectProps) => {
+    const theme = useTheme();
+
+    const handleChange = (event: SelectChangeEvent<typeof selectedOptions>) => {
+        const { target: { value } } = event;
+        setSelectedOptions(
+            // On autofill we get a stringified value.
+            typeof value === 'string' ? value.split(',') : value,
+        );
+    };
+
+    return (
+        <FormControl sx={{ m: 1, width: 300 }}>
```

Comment:
cool, but this is kind of convention breaking

but - I don't really care and I honestly think they won't either, so leave it be

Reply:
copied from google :)